build-tools/xaprepare/xaprepare/ConfigAndData/Dependencies/Linux.DebianCommon.cs

```diff
@@ -10,7 +10,9 @@ abstract class LinuxDebianCommon : Linux
 			new DebianLinuxProgram ("automake"),
 			new DebianLinuxProgram ("cmake"),
 			new DebianLinuxProgram ("build-essential"),
+			new DebianLinuxProgram ("cli-common-dev"),
 			new DebianLinuxProgram ("curl"),
+			new DebianLinuxProgram ("devscripts"),
 			new DebianLinuxProgram ("gcc"),
 			new DebianLinuxProgram ("g++"),
 			new DebianLinuxProgram ("g++-mingw-w64"),
```

Comment:
It would be nice if we could make this dependency optional, based on whether or not the Windows cross-build targets are enabled

Reply:
I agree this would be nice to have, but I'd rather land this without having to rework dependency detection. I added an issue to revisit in the future https://github.com/xamarin/xamarin-android/issues/4173